lib/mix/tasks/ecto.load.ex

```diff
@@ -69,10 +69,12 @@ defmodule Mix.Tasks.Ecto.Load do
         "load structure for #{inspect(repo)}"
       )
 
+      {_repo, source} = Ecto.Migration.SchemaMigration.get_repo_and_source(repo)
```

Comment:
We need to use this `repo`. :) we also need to do similar for ecto.dump!

Reply:
Wait, in dump there is an issue. Since the data is spread out, which of the two repos should we dump? Maybe we should raise if they differ?

Possibly the same issue in load too... perhaps we need to fail in both if the repo changes?

Reply:
ah, thanks :). Ya, I wanted to do the `ecto.dump` change in a difference commit. That change required for `ecto.dump` doesn't seem that obvious to me though, in case you have any other tips that could help :).

Reply:
Well, what do you expect to happen? :) the data is now split in two repositories. I assume in this case they are distinct databases? If so, where should the data be loaded at? In the migration repo or the data repo? Or is it better to raise?

Reply:
:) https://github.com/elixir-ecto/ecto_sql/pull/232/commits/d8c4ff13a3ff8d899a508ccd851a903e25ae008c ??

Reply:
The commit looks good but is that what you really expect? Can you try giving a go at answering the questions above? I think they will provide a better guidance of the approach to take rather than code. :) Thanks!

Reply:
Actually, I think the correct is to load both repos and migrate both repos. I will push a change. :)

Reply:
@sabondano I have pushed two commits, please take a look at them. But we need to test in said cases that we dump two files and load both files. Can you please update the tests according? We are almost there, thank you!

Reply:
@josevalim Thank you! Will updates tests 👍 . In cases where the `repo` and `migration_repo` are the same, we don't want dump and load both of them right? I think we are missing that in the logic within ecto.load and ecto.dump 🤔

Reply:
nvm :).... saw the `Enum.uniq/1` call that accounts for this

Reply:
@josevalim I added dump and load tests for when the `migration_repo` option is used :).